dev/ush/make_ee2_links.sh

```diff
@@ -0,0 +1,421 @@
+#!/bin/bash
```

Comment:
It might be good to have a script like this that isn't specific to the IC folders but more general.  For example if someone downloads an old experiment or GFSv16 (although that has other complications now I guess).  

I'm currently making a new -> old script so I can test if we can reproduce from HPSS or not.  Not sure if this PR is ready to be used or not, so I'm doing it kind of as I run into errors, which is probably not the best way to go about it.  I just found this so I'll probably use some of what's here to add to that.

Reply:
Added in #4247.